### PR TITLE
⚡ Refactor README

### LINKED ISSUE #547: Refactor README
## Problem

Readme here is good but could be better and consise
Could have a gif, centered logo, etc
## Solution

Refactor Readme

[matiassingers/awesome-readme](https://github.com/matiassingers/awesome-readme/blob/master/readme.md) contains some awesome examples

Readme of [amitmerchant1990/electron-markdownify](https://github.com/amitmerchant1990/electron-markdownify/blob/master/README.md) is what I recommend our project can adopt

Waiting for your feedback

### PR DESCRIPTION
**Pull request by nainardev**
fixes #547 

Things @phoqe would need to do
- Add correct Discord link to badge
- Add a screenshot or gif above Features section

### CODE REVIEW

**Comment by phoqe** [comment]
I like the ⚡️ thing, I have some ideas in mind. Will get back to you tomorrow and get the details in place and later merge. Thank you! 😄

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <p align="center">
   <img width="600" src="https://user-images.githubusercontent.com/7033377/77688568-9399c480-6fa0-11ea-9ee2-424a4a99e2e4.png" alt="Logo">
 </p>
-
-RMUIF is short for [React](https://reactjs.org), [Material-UI](https://material-ui.com), and [Firebase](https://firebase.google.com), the three musketeers that make up this template. With [Create React App](https://create-react-app.dev) at its core, you can expect the same tooling to work out of the box. The starting app is essentially a supercharged version of CRA, with all the bells and whistles.
+<h4 align="center">:zap: Supercharged version of CRA, with all the bells and whistles.</h4>
```
Comment (phoqe) [inline README.md:4]: Remove punctuation in the end.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+<p align="center">
+  <a href="discordapp.com"><img src="https://badgen.net/badge/chat/on%20discord/cyan"></a>
```
Comment (phoqe) [inline README.md:6]: Use [shields](https://github.com/badges/shields) instead.

Comment (nainardev) [reply README.md:6]: Traffic to sheild is so high that sometimes badge don't even load

Badgen is new

**Comment by phoqe** [comment]
@nainardev I changed some stuff, take a look.

**Comment by nainardev** [comment]
Two things
 - Tagline is now longer ( important)
 - GitHub emoji codes is safer ( not important)

**Pull request merged**